test/setup_spec.js

```diff
@@ -7,9 +7,9 @@ var apiKey = '1234567abcde';
 
 var scope = nock('https://app.zencoder.com')
               .matchHeader('Zencoder-Api-Key', apiKey)
-              .get('/api/v2/account').times(2)
+              .get('/api/v2/account').times(3)
               .reply(200, scopes.accounts.details)
-              .post('/api/v90/jobs')
+              .post('/api/v90/jobs').times(2)
```

Comment:
Why `/api/v2/` and then `/api/v90`?

Reply:
[There's a test](https://github.com/cgpuglie/zencoder-node/blob/4db79cd43acf9b6c20bc3c26b10524a747534327/test/setup_spec.js#L54) to ensure the 404 error handling returns an error object.